docs/how_to_run.md

```diff
@@ -0,0 +1,187 @@
+
+At the end of this step:
+* If all item submission assets are complete:
+   - A batch folder with complete item submission assets exists in the DSO S3 bucket
+   - Each item submission in the batch is recorded in DynamoDB (with `status="batch_created"`)
+   - **[OPTIONAL]** An email is sent reporting the number of created item submissions. The email includes a CSV file with the batch records from DynamoDB.
+* If any item submission assets were invalid (missing metadata and/or bitstreams):
+   - A batch folder with incomplete item submission assets exists in the DSO S3 bucket
+   - **[OPTIONAL]** An email is sent reporting that zero item submissions were created. The email
+     includes a CSV file describing the failing item submissions with the corresponding error message.
+
+**Data syncing**
+
+✨ If the batch folder was already created (i.e., an S3 bucket in a different deployment environment), DSC can sync the data and avoid repeating data retrieval steps. 
+
+### Queue a batch for ingest
+DSC retrieves the batch records from DynamoDB, and for each item submission, it performs the following steps:
+* Determine whether the item submission should be sent to the DSS input queue
+* Map/transform the source metadata to follow the Dublin Core schema
+* Create and upload a metadata JSON file in the batch folder (under `dspace_metadata/`)
+* Send a message to the DSS input queue
+
+  Note: The message is structured in accordance with the [Submission Message Specification](https://github.com/MITLibraries/dspace-submission-service/blob/main/docs/specifications/submission-message-specification.md).
+
+At the end of this step:
+* Batch records in DynamoDB are updated. Updates are made to the folllowing fields:
+    - `status`: Indicates submit status
+    - `status_details`: Set to error messages (if message failed to send)
+    - `last_run_date`: Set to current run date
+    - `submit_attempts`: Increments by 1
+* **[OPTIONAL]** An email is sent reporting the counts for each submission status. The email includes a CSV file with the batch records from DynamoDB, reflecting the latest information.
+
+### Run DSS
+DSS consumes the submission messages from the input queue in SQS. DSS uses a client to interact with DSpace. For each item submission, DSS reads the metadata JSON file and bitstreams from S3, using the information provided in the message, and creates an item with bitstreams in DSpace.
+
+At the end of this step:
+* Result messages are written to the output queue for DSC (`dss-output-dsc`).
+
+  Note: The message is structured in accordance with the [Result Message Specification](https://github.com/MITLibraries/dspace-submission-service/blob/main/docs/specifications/result-message-specification.md).
+
+### Inspect ingest results
```

Comment:
Very nit picky, very subjective, totally feel free to ignore...

I might propose "Analyze" vs "Inspect".  For reasons I'm having a hard time articulating, "Inspect" feels like it will then _do_ something with what it learns, while "Analyze" feels like it has a more reporting vibe, which feels more accurate.

Reply:
On a second read, "inspect" makes me think we're _looking at the item ingested in DSpace_, and with that, I agree "analyze" somehow feels different. 🤔